drivers/docker/config.go

```diff
@@ -258,7 +258,7 @@ var (
 			"config": hclspec.NewBlockAttrs("config", "string", false),
 		})),
 		"mac_address": hclspec.NewAttr("mac_address", "string", false),
-		"mounts": hclspec.NewBlockSet("mounts", hclspec.NewObject(map[string]*hclspec.Spec{
+		"mounts": hclspec.NewBlockList("mounts", hclspec.NewObject(map[string]*hclspec.Spec{
```

Comment:
I changed `devices` and `mounts` declaration to be a list instead of set, to preserve order for tests.  This change shouldn't be customer facing.